contrib/test/integration/main.yml

```diff
@@ -118,7 +118,7 @@
       include: "build/kubernetes.yml"
       vars:
           force_clone: True
-          k8s_git_version: "master"
+          k8s_git_version: "release-1.10"
```

Comment:
Don't we intend to track k8s' master in our master branch?

Reply:
@wking We will track release-1.10 in master till we make a branch for it next week. At that point we can switch back to master.